### ISSUE TITLE
vertrauensdiensteanbieter bei ausgefallener sicherheitseinrichtung

### ISSUE THREAD

**Issue by eczach** [a-sit-plus/at-registrierkassen-mustercode#281]
folgende frage:
die sicherheitseinrichtung ist ausgefallen, die karte kann nicht gelesen werden, somit ist auch nicht definiert, welches kürzel für den vertrauensdiensteanbieter verwendet werden soll (es könnten ja z.b für die kassa ja auch karten zweier VDA's angemeldet sein - z.b. AT1 und AT2).
was sollte dann eingetragen werden? das prüftool verlangt jedenfalls hier eine angabe.

**Comment by Flanelli** [comment]
Das wird m.E. wohl oder übel einerlei sein aber am einfachsten wär wohl den Wert des zuletzt funktionierenden Beleges heranzuziehen. Nachdem ja auch mehrere Zertitikate bzw.Anbieter zulässig sind kann es wohl auch keinen Zwang geben einen ganz bestimmten in diesem Fall zu wählen.

**Comment by eczach** [comment]
denke dies auch - es wird ja anscheinend auch verpflichtend im prüftool die angabe einer zertifikatsnummer verlangt, die für die kassa angemeldet ist. so funktionieren die prüfungen - aber ist das wirklich so vorgesehen? in den dokumenten habe ich nichts gefunden.

**Comment by ErichFreitag** [comment]
Das gleiche Problem haben sie ja auch mit der Seriennummer. Da ist die zuletzt verwendete Seriennummer zu verwenden und die dazu passende VDA-ID. Sie können auch eine andere (zugeordnete) Seriennummer plus VDA-ID aus einer früheren Signierung verwenden - die mögliche Gefahr ist, dass es dieses Zertifikat vielleicht nicht mehr gibt. 

"So vorgesehen" ist es nicht weil wahrscheinlich dieser Fall nicht im Detail berücksichtigt wurde, es sich jedoch um die akzeptierte und vernünftigste Vorgangsweise handelt (vergleiche https://github.com/a-sit-plus/at-registrierkassen-mustercode/issues/103 und andere).

**Comment by eczach** [comment]
@ErichFreitag "Da ist die zuletzt verwendete Seriennummer zu verwenden und die dazu passende VDA-ID" - ist das irgendwo so festgelegt? 
ansonsten würde ich davon ausgehen, dass mangels einer regelung und aufgrund des verhaltens des prüftools die seriennummer eines dem unternehmer zugeordneten und aktuell anmeldeten zertifikats verwendet werden muss.

**Comment by ErichFreitag** [comment]
Nein, es ist eben nicht festgelegt, aber "best practice", siehe oben verlinkte Issues und zugehörige.

Im Grunde ist es korrekt, irgendein zugeordnetes Zertifikat zu verwenden. Die Frage ist nur, ob sie a) darauf zugreifen können (dann könnten sie ja auch das anstelle des ausgefallenen verwenden) und b) wie sie sicher wissen, dass es noch angemeldet ist. Daher ist die Verwendung des letzten am einfachsten und am sichersten.

**Comment by eczach** [comment]
die best practice-lösung hat auch ihre probleme - folgendes szenario:
ein unternehmer mit nur einer angemeldeten signaturkarte und einer kassa. die signaturkarte geht defekt oder wird verloren und wird deshalb bei FON abgemeldet. damit können aber auf der kassa - auch mit "sicherheitseinrichtung ausgefallen" keine belege mehr erstellt werden, da diese bei der belegprüfung bei FON nicht mehr akzeptiert wird - eigentlich ist damit die kassa ausgefallen.
wenn aber die verlorene karte NICHT bei FON abgemeldet wird, können belege erstellt werden - aber das ist ein eher schlamperter ansatz :-)
für mich wäre der passenste ansatz, wenn bei ausgefallener signatureinrichtung ein klar definiertes vertrauensdienste-kennzeichen und eine definierte zertifikats-seriennummer (z.b.0) angegeben werden könnte (und keine beliebige angemeldete karte) - aber vielleicht übersehe ich hier etwas bzw. gibt es hier doch noch andere überlegungen und regelungen?

**Comment by ErichFreitag** [comment]
Die Idee ist ja nicht schlecht, aber ist halt nicht vorgesehen.

Man muss es nur richtig machen. Also die defekte Karte dann abmelden, wenn man die neue anmeldet. Das ist ein Beispiel dafür, warum die RKSV nicht nur technische, sondern auch organisatorische Auswirkungen mit sich bringt.

Wenn jemand etwas defektes nicht gesetzeskonform wieder in Ordnung bringt, fällt es halt spätestens bei der nächsten (Jahres-)Belegprüfung auf.

**Comment by asitplus-pteufl** [comment]
Technische Ergänzung: Wenn die Sig-Einrichtung ausgefallen ist, wissen Sie ja welche das ist, und nehmen die Seriennummer, ZDA-ID davon. Sie müssen es ja wissen, sonst können Sie sie ja auch nicht ordnungsgemäß in FinOnline als ausgefallen melden (nach 48 Stunden). Wenn eine funktionsfähige vorhanden ist ergibt sich das Problem sowieso nicht, dann wird einfach diese verwendet.

@eczach ja, das ist ein Punkt, allerdings ist es ja dann gleich als ob Sie gar keine Sig-Einrichtung registriert hätten, sprich Sie verwenden eine Kasse für die es keine gültigen Sig-Einrichtungen gibt. Das Problem das sie oben nennen ist natürlich relevant, ich kann das nur an das BMF weiterleiten.
